{{cookiecutter.github_repository}}/provisioner/roles/nginx/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 
 # running cron job each monday
 - name: add letsencrypt renew cron
-  cron: name="Renew all certificates" minute="0" hour="2" weekday="1" job="PATH=$PATH:/usr/local/bin && certbot-auto renew --webroot -w {{ letsencrypt_challange_root }}  >> /var/log/certbot.log"
+  cron: name="Renew all certificates" minute="0" hour="2" weekday="1" job="PATH=$PATH:/usr/local/bin && certbot-auto renew --non-interactive --webroot -w {{ letsencrypt_challange_root }}  >> /var/log/certbot.log"
```

Comment:
lets add `&& service nginx reload` as well here we need to reload nginx when the cert gets updated.